pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/index.md

```diff
@@ -0,0 +1,531 @@
+---
+layout: layout.pug
+navigationTitle: Air-Gapped Bundles
+title: Air-Gapped Bundles
+menuWeight: 10
+excerpt: Download and install air-gapped bundles when provisioning in an air-gapped environment.
+beta: false
+enterprise: false
+---
+
+## Review the air-gapped bundle contents
+
+The air-gapped bundle contains the necessary OS dependencies and images required to set up a DKP Essential installation in an air-gapped environment. The download and installation of these bundles is agnostic to a specific environment, and is compatible with any of the environments listed in the table below:
+
+| Operating System      | Kernel                      | Air-Gapped     |
+|-----------------------|-----------------------------|----------------|
+| [CentOS 7.9][centos7] | 3.10.0-1160.el7.x86_64      | Yes            |
+| [RHEL 7.9][rhel_7_9]  | 3.10.0-1160.el7.x86_64      | Yes            |
+
+The air-gapped bundle includes the following packages and tooling:
+
+- Command line tooling
+- Konvoy air-gapped bundle,
+- Kommander air-gapped bundle,
+- Konvoy Image Builder
+- CentOS/RHEL 7.9 dependencies
+- Setup script
+- Cluster initialization script
+- Docker registry image
+
+## Download and expand  the air-gapped bundle
+
+
+1.  Define an environment variable for the Kubernetes version that corresponds with Konvoy release you are installing. You can find the correct Kubernetes version by checking the release notes for the release you are installing.
+
+    ```bash
+    export VERSION=1.21.6
+    ```
+
+1.  Download the air-gapped bundle files.
+
+    ```bash
+    curl -O downloads.d2iq.com/dkp/v2.1.1/dkp_airgapped_bundle_v2.1.1_linux_amd64.tar.gz
+    curl -O downloads.d2iq.com/konvoy/airgapped/pip-packages.tar.gz
+    ```
+1.  Transfer the Air-gapped bundle to the Bootstrap host.
+
+    ```bash
+	scp dkp_airgapped_bundle_v2.1.1_linux_amd64.tar.gz <user>@<bastion_host>:~/
+	```
+1.  Login to the Bastion host using the same credentials used in the previous step.
+
+	```bash
+	ssh <user>@<bastion_host>
+	```
+
+1.  Expand the air-gapped bundle artifact.
+
+
+```bash
+
+    tar -xvf dkp_airgapped_bundle_v2.1.1_linux_amd64.tar.gz
+    ```
+1.  Move into the created directory.
+
+
+  ```bash
+
+    cd dkp-v2.1.1
+    ```
+## Prepare the bootstrap node and environment
+
+Run the setup command in order to load the bootstrap:
+  * Install Docker engine
+  * Load and create local registry
+  * Update Docker daemon configuration with registry reference
+  * Seed local registry
+  * Load bootstrap image required for DKP CLI
+
+1.  Run setup.
+
+    ```sh
+    sudo ./setup
+    ```
```

Comment:
What is this `setup` command/script?

Reply:
Probably came from John Miller I'm guessing.